File: The-Food-System-main/1- Server/TalabatReplica/ECommerce.DAL/Models/IdentityModels/RegisterModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ECommerce.DAL.Models.IdentityModels
{
    public class RegisterModel
    {
        [Required, MaxLength(15)]
        public string FirstName { get; set; }

        [Required, MaxLength(15)]
        public string LastName { get; set; }

        [Required, MaxLength(50)]
        public string Username { get; set; }

        [Required, StringLength(128)]
        [DataType(DataType.EmailAddress)]
        public string Email { get; set; }

        [Required, StringLength(256)]
        [DataType(DataType.Password)]
        public string Password { get; set; }

        public bool AdminCheck { get; set; } 

    }
}

File: The-Food-System-main/2-Client/TalabatReplica/src/app/Models/register.model.ts
export class Register {
    firstName:string|null="";
    lastName: string|null="";
    username:string|null="";
    email: string|null="";
    password: string|null="";
    AdminCheck:boolean|null=false

    constructor(firstName:string|null ,lastName:string|null, username:string|null, email:string|null, password: string|null, AdminCheck:boolean|null)
    {
        this.firstName=firstName;
        this.lastName=lastName;
        this.username=username;
        this.email=email;
        this.password=password;
        this.AdminCheck=AdminCheck;

    }

}
